bin/cli.js

```diff
@@ -112,11 +112,9 @@ prettyCLI
         } else {
             gscan.check(argv.themePath, cliOptions)
                 .then(theme => outputResults(theme, cliOptions))
-                .catch(function ENOTDIRPredicate(err) {
-                    return err.code === 'ENOTDIR';
-                }, function (err) {
-                    ui.log(err.message);
-                    ui.log('Did you mean to add the -z flag to read a zip file?');
+                .catch((err) => {
+                    ui.log(`\n${chalk.red(err.message)}`);
+                    err.code === 'ENOTDIR' && ui.log('Did you mean to add the -z flag to read a zip file?');
```

Comment:
Would be preferable to avoid the `&&` shourtcut notation, it is pretty hard to read unless one knows how JS works. The simpler the better - regular `if` works just fine in this situation 😄